sass/public.scss

```diff
@@ -0,0 +1,237 @@
+@import '/sass/variables.scss';
+@import "../node_modules/bootstrap/scss/mixins";
+@import "../node_modules/bootstrap/scss/functions";
+@import "../node_modules/bootstrap/scss/variables";
+$html-font-size: 16px;
+@function stripUnit($value) {
+    @return $value / ($value * 0 + 1);
+}
+
+@function toRem($pxValue) {
+    @return #{stripUnit($pxValue) / stripUnit($html-font-size)}rem;
+}
+
+
+/**********************************/
+
+
+/*********** Nav Bar CSS **********/
+
+
+/**********************************/
+
+section {
+    min-height: toRem(300px)
+}
+
+.navbar {
+    position: relative;
+    transition: .5s;
+    z-index: map-get($z-index, nav);
+    a {
+        text-decoration: unset;
+    }
+    &__navbar {
+        &-collapse {
+            display: flex;
+            -ms-flex-preferred-size: auto;
+            flex-basis: auto;
+        }
+    }
+    &__navbar-brand {
+        color: $orangeColor;
```

Comment:
bad colors naming, we shouldn't use the color name instead we should rely on branding naming conversion like primary/ secondary see https://material.io/design/color/the-color-system.html#color-theme-creation